src/main/java/org/example/homework14/AccordionInteractionTest.java

```diff
@@ -0,0 +1,29 @@
+package org.example.homework14;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class AccordionInteractionTest {
```

Comment:
Don't put test classes to `main/java/...` no one will expect to look for tests in main
and according to the task for Selenium part you should create a new project